test/babashka/cli_test.cljc

```diff
@@ -184,10 +184,11 @@
                       :coerce []
                       :default ["src" "test"]
                       :default-desc "src test"}}]
-    (is (= (str/trim "  -i, --from   <format> edn      The input format. <format> can be edn, json or transit.
+    (is (= (str/trim "
+  -i, --from   <format> edn      The input format. <format> can be edn, json or transit. 
   -o, --to     <format> json     The output format. <format> can be edn, json or transit.
-      --paths           src test Paths of files to transform.
-  -p, --pretty                   Pretty-print output.")
+      --paths           src test Paths of files to transform.                            
```

Comment:
I'm not sure about the whitespace on the right, probably we can get rid of this. Other tools like `grep -h` don't emit right-side whitespace either.